api.js

```diff
@@ -28,27 +28,17 @@ function Api(files, options) {
 	this.options = options || {};
 	this.options.require = (this.options.require || []).map(resolveCwd);
 
-	if (!files || files.length === 0) {
-		this.files = [
-			'test.js',
-			'test-*.js',
-			'test'
-		];
-	} else {
-		this.files = files;
-	}
+	Object.keys(Api.prototype).forEach(function (key) {
+		this[key] = this[key].bind(this);
+	}, this);
+
+	this._reset();
```

Comment:
If you look at the overall diff you'll see that these lines were moved above `this.excludePatterns`. Would be nice to leave them undisturbed.